tinyxml2.cpp

```diff
@@ -473,10 +473,10 @@ const char* XMLUtil::GetCharacterRef( const char* p, char* value, int* length )
     *length = 0;
 
     if ( *(p+1) == '#' && *(p+2) ) {
-        unsigned long ucs = 0;
-        TIXMLASSERT( sizeof( ucs ) >= 4 );
+        unsigned long long ucs = 0;

```

Comment:
These should be `stdint` types, not long long. Also - why the change? How is overflow being addressed?